### DISCUSSION TITLE
Neuroscope2 added event intervals

### DISCUSSION THREAD

**Discussion by ryanharvey1** [petersenpeter/CellExplorer#87] [Ideas]
Hi Peter,

In Neuroscope2, it is possible to add events, but it does not seem to have the option to add intervals to these events. Is it possible for this to be implemented in a future commit?

Thank you,
Ryan

**Comment by petersenpeter** [comment]
Hi Ryan 

I can take a look at this during the holidays. It should not be too complicated to implement. 

/ Peter

**Comment by petersenpeter** [comment]
You can now add intervals to events. I added two buttons to the left side menu in the events tab group `+ interval` and `- interval`. The intervals are added as a new field to the events struct `.added_intervals`. You can also adjust and remove intervals with `- interval`. Both are done by defining the boundaries of events with your left mouse, finishing with a right mouse click, and removing the last border with a middle click. 

Let me know if you experience any issues!

**Comment by ryanharvey1** [comment]
Hi Peter,

I tested out the new interval functionality, and it's working well!

Thanks for developing this feature.

Best,
Ryan
